src/first_steps/compilation_portability.md

```diff
@@ -1,13 +1,13 @@
 ## Compilation and Portability
 
-Currently the core of rizin can be compiled on many systems and architectures, but the main development is done on GNU/Linux with GCC, and on MacOS X with clang. Rizin is also known to compile on many different systems and architectures (including TCC and SunStudio).
+The core of Rizin can be compiled on various systems and architectures, with primary development conducted on GNU/Linux using GCC and on MacOS X with clang. Rizin is known to compile on a wide range of systems and architectures, including compatibility with TCC and SunStudio.
```

Comment:
s/The core of Rizin/Rizin. Also remove "conducted on Linux/macOS" and replace with "conducted on UNIX systems (Linux, macOS, etc)" or something like that.

Reply:
TCC (TinyCC), maybe a link to their repo.

Reply:
Also, I am not sure about SunStudio. Also Oracle uses Clang nowadays, I think. Just remove its mention completely.